site/index5.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Katy</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <link href='https://fonts.google.com/specimen/Roboto' rel='stylesheet' type='text/css'>
+</head>
+<body>
+<header  >
+    <div style=" background-color:rgb(73, 68, 68,01) ; height:40px; line-height: 30px;letter-spacing:1px; word-spacing:10px;padding-left: 10px; width: 100%;margin: 0;font-size: 100%;"> 
```

Comment:
I think this `<div>` is redundant. You can remove it and set all this css properties to the `<header>`